spec/controllers/presentations_controller_spec.rb

```diff
@@ -0,0 +1,110 @@
+# spec/controllers/presentations_controller_spec.rb
+require 'rails_helper'
+
+RSpec.describe PresentationsController, type: :controller do
+  describe 'GET #index' do
+    subject { get :index }
+
+    it 'renders the template' do
+      expect(subject).to render_template :index
+    end
+  end
+
+  describe 'GET #show' do
+    subject { get :show, params: { id: params } }
+
+    let(:params) { FactoryGirl.create(:presentation) }
+
+    it 'renders the template' do
+      expect(subject).to render_template :show
+    end
+  end
+
+  describe 'GET #new' do
+    subject { get :new }
+
+    it 'renders the template' do
+      expect(subject).to render_template :new
+    end
+  end
+
+  describe 'GET #edit' do
+    subject { get :edit, params: { id: params } }
+
+    let(:params) { FactoryGirl.create(:presentation) }
```

Comment:
Looks like we are running this same line twice in your tests. Could be done once between lines four and five.